cloudinit/config/cc_growpart.py

```diff
@@ -284,11 +284,15 @@ def resize(self, diskdev, partnum, partdev):
 
 
 def get_size(filename):
```

Comment:
Let's add an `Optional[int]` return type here since we're changing the return type